### PR TITLE
Layers neophyten

### PR DESCRIPTION
**Pull request by alcapat**

### CODE REVIEW

**Comment by loicgasser** [comment]
legend img background not transparent
if wms based image legends you can simply use:
```make legends BODID=ch.foo```

**Comment by loicgasser** [comment]
convert test.png -transparent white transparent.png

**Comment by alcapat** [comment]
legend img background is now transparent

File: ltalp_run.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```
Comment (loicgasser) [inline ltalp_run.sh]: please remove that

**Comment by loicgasser** [comment]
LGTM please remove the following files:

results.csv
scripts/results.csv
topics.png
ltalp_run.sh

**Comment by alcapat** [comment]
Jenkins test

**Comment by loicgasser** [comment]
for tomorrow?

**Comment by loicgasser** [comment]
please rebase also

**Comment by AFoletti** [comment]
There is no data deploy planned for tomorrow. Deploy of those layers on 11.10

**Pull request merged**